Language/Java/Parser.hs

```diff
@@ -87,16 +87,6 @@ import Text.ParserCombinators.Parsec.Error
 import Prelude hiding (exp, (>>), (>>=))
 import qualified Prelude as P ((>>), (>>=))
 
-#if __GLASGOW_HASKELL__ < 707
-import Control.Applicative ( (<$>), (<$), (<*) )
--- Since I cba to find the instance Monad m => Applicative m declaration.
-(<*>) :: Monad m => m (a -> b) -> m a -> m b
-(<*>) = ap
-infixl 4 <*>
-#else
-import Control.Applicative ( (<$>), (<$), (<*), (<*>) )
-#endif
-
```

Comment:
Vermutlich ist es in Ordnung, den Part zu entfernen. Ich hatte mich bisher nicht dazu informiert.
Hattest du das mit Jan abgesprochen?

Reply:
Kann mir nicht erklären, wieso das entfernt wurde. Ich füge es wieder hinzu.